sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientLiveTests.cs

```diff
@@ -186,8 +187,10 @@ public async Task DetectLanguageBatchWithErrorTest()
             Assert.IsTrue(!results[0].HasError);
             Assert.IsTrue(!results[2].HasError);
 
+            var exceptionMessage = "Cannot access result for this document, due to error InvalidDocument: Document text is empty.";
```

Comment:
In Java, we have included the document ID too in the message, that could help the user know which document exactly returned an error.

Reply:
ID added